mne_bids_pipeline/_config.py

```diff
@@ -426,9 +422,40 @@
 `'error'` to suppress warnings emitted by read_raw_bids.
 """
 
-###############################################################################
-# BREAK DETECTION
-# ---------------
+plot_psd_for_runs: Union[Literal["all"], Sequence[str]] = "all"
+"""
+For which runs to add a power spectral density (PSD) plot to the generated
+report. This can take a considerable amount of time if you have many long
+runs. In this case, specify the runs, or pass an empty list to disable raw PSD
+plotting.
+"""
+
+random_state: Optional[int] = 42
+"""
+You can specify the seed of the random number generator (RNG).
+This setting is passed to the ICA algorithm and to the decoding function,
+ensuring reproducible results. Set to `None` to avoid setting the RNG
+to a defined state.
+"""
+
+shortest_event: int = 1
+"""
+Minimum number of samples an event must last. If the
+duration is less than this, an exception will be raised.
+"""
+
+# %%
+# # Preprocessing
+
+# ## Break detection
+#
+# ---
+# tags:
+#   - preprocessing
+#   - artifact-removal
+#   - raw
+#   - events
+# ---
```

Comment:
@hoechenberger this is my one open question -- useful to have the `tags` in the `_config.py` or no? If not I'll move to the parsing `.py` file instead.

Reply:
How would you add them to the Python files? What would that look like?

Reply:
In the parsing gen file something like
```
tags = {
    "break detection": ("preprocessing", "artifact-removal", "raw", "events"),
    ...
```
One tuple of tags per option category, then the parser adds them to the .md files in the right place

Reply:
I don't have a clear preference here, I suppose I'm fine with either

Reply:
Thinking about it more I decided to move it to `gen_settings.py` -- that way someday we could consider auto-generating / adding tags. And I think it keeps `_config.py` a little bit more compact which is nice.